### PR TITLE
openvpn 2.4.0

### PR DESCRIPTION
**Pull request by vmbracvlvm**

### CODE REVIEW

File: recipes/openvpn/openvpn_2.4.0.oe [hunk 1]
```diff
@@ -0,0 +1,6 @@
+require ${PN}.inc
+
+EXTRA_OECONF += "--disable-plugin-auth-pam"
+
+EXTRA_OECONF += "IPROUTE=${base_sbindir}/ip --enable-iproute2"
```
Comment (esben) [inline recipes/openvpn/openvpn_2.4.0.oe]: Do we really need to specify IPROUTE?

Comment (vmbracvlvm) [inline recipes/openvpn/openvpn_2.4.0.oe]: In my experience yes. If not set configure will try to do it.

Comment (esben) [inline recipes/openvpn/openvpn_2.4.0.oe]: Ouch, so ./configure will look it up on host, and build in that path?

**Comment by PrevasJsch** [comment]
Actually it seems that specifying IPROUTE is only needed when building in my customized manifest. When building against oe-lite master branches it seems to work.

**Pull request merged**